docs/news/posts/2024-07-30-new-maswe.md

```diff
@@ -0,0 +1,118 @@
+---
+title: "Introducing the new Mobile App Security Weakness Enumeration (MASWE)"
+date: 2024-07-30
+authors: [carlos]
+slug: new-maswe
+---
+
+The OWASP MAS project continues to lead the way in mobile application security, providing robust and up-to-date resources for developers and security professionals alike. Our team has been working diligently with the MAS community and industry to refactor the [Mobile Application Security Verification Standard (MASVS)](https://mas.owasp.org/MASVS/) and the [Mobile Application Security Testing Guide (MASTG)](https://mas.owasp.org/MASTG/). In this blog post, we'll walk you through our latest addition to the MAS project: the brand new [Mobile App Security Weakness Enumeration (MASWE)](https://mas.owasp.org/MASWE/).
+
+<center>
+<img style="width: 40%; border-radius: 5px" src="/assets/maswe-overview.png"/>
+</center>
+
+## Refactoring the MASTG
+
+We began the refactoring process in 2021, focusing first on the MASVS and then on the MASTG. Our primary goal was to break the MASTG v1 into modular components, including tests, techniques, tools, and applications.
+
+<center>
+<img style="width: 80%; border-radius: 5px" src="/assets/news/mas_v2_arch.png"/>
+</center>
+
+This modular approach allows us to **maintain and update each component independently**, ensuring that the MASTG remains current and relevant. For example, in our previous structure, the MASTG consisted of large test cases within a single markdown file. This was not only difficult to maintain but also made it challenging to reference specific tests; and it was impossible to have metadata for each test.
+
+The new structure divides tests into individual pages (Markdown files with metadata), each with its own ID (`MASTG-TEST-****`) and links to relevant techniques (`MASTG-TECH-****`) and tools (`MASTG-TOOL-****`). This encapsulation ensures that **each test is easily referenced and promotes reusability** across all MAS components. For example, you can now see which tools are used in a particular test and which techniques are applied. This is currently partially implemented, but when we're done, you'll be able to do many interesting things, like see all tests that use a particular tool or technique.
```

Comment:
Second try. Hopefully it will be clearer this time.

```suggestion
The new structure divides tests into individual pages (Markdown files with metadata), each with its own ID (`MASTG-TEST-****`) and links to relevant techniques (`MASTG-TECH-****`) and tools (`MASTG-TOOL-****`). This encapsulation ensures that **each test is easily referenced and promotes reusability** across all MAS components. For example, you can open a test and see what tools and techniques are being used, and soon you'll be able to do the same in reverse: open a tool or technique and see all the tests that use it. This deep cross-referencing can be extremely powerful when exploring the MASTG.
```